name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RenaultBankDirektPDFExtractor.java

```diff
@@ -0,0 +1,169 @@
+package name.abuchen.portfolio.datatransfer.pdf;
+
+import java.util.Map;
+import java.util.function.BiConsumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Block;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.DocumentType;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Transaction;
+import name.abuchen.portfolio.model.AccountTransaction;
+import name.abuchen.portfolio.model.Client;
+
+@SuppressWarnings("nls")
+public class RenaultBankDirektPDFExtractor extends AbstractPDFExtractor
+{
+    static final String DEPOSIT_2019 = "^(?<date>\\d+.\\d+.)  (.*-Gutschrift)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
+    static final String REMOVAL_2019 = "^(?<date>\\d+.\\d+.)  (Internet-Euro-Überweisung)(\\s*)(?<amount>[\\d\\s,.]*)(\\-)(.*)";
+    static final String INTEREST_2019 = "^(?<date>\\d+.\\d+.)  (.*)(Zinsen\\/Kontoführung)(\\s*)(?<amount>[\\d\\s,.]*)(\\+$)";
+    
+    static final String DEPOSIT_2021 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (.*gutschr\\.?)(\\s*)(?<amount>[\\d\\s,.]*) [H]";
+    static final String REMOVAL_2021 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (Umbuchung)(\\s*)(?<amount>[\\d\\s,.]*) [S]";
+    static final String INTEREST_2021 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (Abschluss)(\\s*)(?<amount>[\\d\\s,.]*) [H]";
+    
+    static final String CONTEXT_KEY_YEAR = "year";
+    static final String CONTEXT_KEY_CURRENCY = "currency";
+
+    public RenaultBankDirektPDFExtractor(Client client)
+    {
+        super(client);
+
+        addBankIdentifier("305 200 37"); //$NON-NLS-1$
+        addBankIdentifier("Renault Bank direkt"); //$NON-NLS-1$
+        
+        addTagesgeldTransactionWith2019Format();
```

Comment:
in englisch, wenn möglich :-)

Reply:
Ich seh das eher als Domänenobjekt, welches hier in einem deutschen Kontext befindet.
Diese Methode des Extractor erledigt, zu diesem Zeitpunkt, nur die Tagesgeld-Tranaktionen und zwar die im deutschsprachigen PDF-Dokumenten. 

Im Englischen scheint es hier auch keine wirklich tolle Übersetzung zu geben. "Instant access savings account" ist schon sehr sperrig und ich wage zu bezweifeln, dass das jemand versteht.
Auf Teufel komm raus Dinge zu übersetzen ist ein Fehler den ich an anderen Stellen schon häufiger gesehen habe und immer wieder zu großen Verständnisproblemen und Folgefehlern geführt hat. 

Als Kompromiss habe ich hier jetzt aber einfach das "Tagesgeld" komplett entfernt.